Mapper/Mapper.swift

```diff
@@ -342,6 +361,7 @@ public struct Mapper {
 
      - returns: The object for the given key
      */
+    @warn_unused_result
     private func JSONFromField(field: String) -> AnyObject? {
```

Comment:
I think we should leave it on this one? Although it should be obvious we don't want people to use this incorrectly in the Mapper implementation either?